app/.gitignore

```diff
@@ -28,6 +28,8 @@
 /web/fileadmin/_processed_/**/*
 /web/uploads/**/*
 
+# remove the cssmapping files
+*.map
```

Comment:
Would you think it makes sense to ignore them globally, or just within the dist folder (something like `app/web/typo3conf/ext/theme/Resources/Public/SourceMaps/**/*`)